_posts/blog/2021-04-08-whitakerlab-at-collabw21.md

```diff
@@ -0,0 +1,71 @@
+---
+layout: post
+title: The WhitakerLab at CollabW21
+categories: blog
+excerpt: Notes from the Lab's feedback about the SSI organised event
+tags: [lab-meeting] ### INSERT TAGS IF APPLICABLE
+image:
+  feature:
+link:
+date: 2021-04-08 ### UPDATE date
+modified:
+share: true
+author: sarah_gibson ### CHANGE to your author name (in _data/authors.yml)
+---
+
+At the end of March 2021, many members of the Whitaker Lab attended the Software Sustainability Institute's Collaborations Workshop.
+At the following lab meeting, those who attended shared their experiences with the rest of the lab.
+This is a blog post covering the event and the lab member's reflections from it.
+
+## What is the SSI?
+
+The [Software Sustainability Institute](https://software.ac.uk/about) (SSI for short) is a network of UK universities, Research Software Engineering groups and policy makers dedicated in improving the quality, sustainability and recognition of research software.
```

Comment:
I might have said "dedicated _to_", but perhaps that's more of a stylistic thing?

Reply:
Nope, I think that's an artefact from chopping and changing stuff in a previous review - good catch!

Reply:
```suggestion
The [Software Sustainability Institute](https://software.ac.uk/about) (SSI for short) is a network of UK universities, Research Software Engineering groups and policy makers dedicated to improving the quality, sustainability and recognition of research software.
```